src/edit/actions/RestoreAction.js

```diff
@@ -1,7 +1,7 @@
 L.RestoreAction = L.EditAction.extend({
   initialize(map, overlay, options) {
-    var mode = L.Utils.getNestedVal(overlay, 'editing', '_mode');
-    var edited = overlay.edited;
+    const mode = L.Utils.getNestedVal(overlay, 'editing', '_mode');
```

Comment:
Hi, I think this one should be `let` because the mode could change -- if you're in editing mode or not, does that make sense? 
```suggestion
    let mode = L.Utils.getNestedVal(overlay, 'editing', '_mode');
```

Reply:
Hi, @jywarren okay that makes perfect sense i was unsure about it at first. Should I make changes to it and push it again?